pyproject.toml

```diff
@@ -155,6 +168,8 @@ ignore = [
     "D205",    # our documentation style allows a folded first line
     "EM101",   # justification (duplicate string in traceback) is silly
     "EM102",   # justification (duplicate string in traceback) is silly
+    "FBT001",  # positional booleans are normal for Pydantic field defaults
+    "FBT002",  # positional booleans are normal for Pydantic field defaults
```

Comment:
I haven't been suppressing these; I think they're good diagnostics. When initializing a model, you can use keyword arguments, and I think you always should at least for boolean fields because otherwise it's inobvious what the boolean means.

Reply:
Oh, OK, I was misreading this as "don't use booleans" rather than "if you use a boolean make sure it's keyword-only".